simgui.json

```diff
@@ -26,8 +26,36 @@
   "NTProvider": {
```

Comment:
Intake put simgui*.json into .gitignore so to remove the merge conflicts, remove this file from the commit.